shell/src/main/java/org/apache/accumulo/shell/commands/summaryReport/SummaryReport.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.accumulo.shell.commands.summaryReport;
+
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+import java.util.ArrayList;
+import java.util.EnumSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Set;
+import java.util.TreeMap;
+import java.util.TreeSet;
+
+import org.apache.accumulo.fate.AdminUtil;
+import org.apache.accumulo.fate.ReadOnlyTStore;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+public class SummaryReport {
```

Comment:
I like the separate classes but it is confusing to have a `summaryReport` package when that is not the command. Maybe include them in the same package as FateCommand? Or Create a new package called `fate`?  If you don't want to move `FateCommand` you could just prefix the new classes with `FateCommand`.

Reply:
I posted a change renaming the package to `fateCommand` and added `Fate`  so that its `FateSummaryReport` and `FateTxnDetails`

I didn't move FateCommand itself - there seems to be some value having all of the commands at the same level.

@milleruntime - Better?